Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM openjdk:11 as builder
 
 WORKDIR /cloudwatch_exporter
 ADD . /cloudwatch_exporter
-RUN apt-get -qy update && apt-get -qy install maven && mvn package && \
+RUN apt-get -qy update && apt-get -qy install maven && mvn clean package && \
```

Comment:
Why do you need the clean?

Reply:
I don't really know why but the Docker build seemed to sometimes use outdated code when I did not use "clean". We can remove this though, I kind of accidentally committed this.